mapzen-places-api/src/main/java/com/mapzen/places/api/internal/PeliasFeatureToPlaceConverter.java

```diff
@@ -0,0 +1,60 @@
+package com.mapzen.places.api.internal;
+
+import com.mapzen.pelias.gson.Feature;
+import com.mapzen.places.api.LatLng;
+import com.mapzen.places.api.LatLngBounds;
+import com.mapzen.places.api.Place;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * Handles converting Pelias {@link Feature} objects into {@link Place} objects for
+ * {@link PeliasCallbackHandler}.
+ */
+class PeliasFeatureToPlaceConverter {
+
+  private PeliasLayerToPlaceTypeConverter layerConverter;
+
+  /**
+   * Constructor.
+   */
+  PeliasFeatureToPlaceConverter() {
+    layerConverter = new PeliasLayerToPlaceTypeConverter();
+  }
+
+  /**
+   * Construct a {@link Place} object from the given feature.
+   * @param feature
+   * @return
+   */
+  Place getFetchedPlace(Feature feature) {
+    //TODO: fill in missing values
```

Comment:
Added info in javadocs and am throwing an exception now when those fields are attempted to be accessed. I'll also update the documentation PR.